WMF Framework/PageInfoFetcher.swift

```diff
@@ -0,0 +1,43 @@
+@objc(WMFPageInfoFetcher)
+public final class PageInfoFetcher: Fetcher {
```

Comment:
Is there a reason the namespace returned by `/page/summary` via the `ArticleSummaryController` or `ArticleSummaryFetcher` couldn't be used?

Reply:
It's a heavier call that fetches more data that's needed for this case

Reply:
It's likely the summary will be requested later anyway, requesting it here and having it cached by the networking layer for that later call is likely lighter weight overall. Also, `/page/summary` is cached server-side as well, I don't think the action API is

Quick unscientific benchmark shows that the two requests take about the same amount of time:
```
/Users/wmf/source/wikipedia-ios % for i in {1..10}; curl -s -w "%{time_total}\n" -o /dev/null -H "Accept: application/json; charset=utf-8; profile=\"https://www.mediawiki.org/wiki/Specs/Summary/1.1.2\"" -H "Accept-Encoding: gzip" https://en.wikipedia.org/api/rest_v1/page/summary/Barack_Obama
0.147791
0.136603
0.158135
0.156236
0.145728
0.156962
0.156832
0.158531
0.145259
0.196570
/Users/wmf/source/wikipedia-ios % for i in {1..10}; curl -s -w "%{time_total}\n" -o /dev/null -H "Accept: application/json; charset=utf-8;" -H "Accept-Encoding: gzip" "https://en.wikipedia.org/w/api.php?action=query&format=json&titles=Barack_Obama"
0.198058
0.186314
0.187906
0.192104
0.189743
0.184776
0.176015
0.196304
0.185752
0.187870
```